test/BF16/mlp-single-layer-blocked-bf16.mlir

```diff
@@ -42,7 +42,7 @@
         %f1 = arith.extf %v0:vector<4x4xbf16> to vector<4x4xf32>
         
        //
-       // CHECK:( ( 129, 129, 129, 129 ), ( 129, 129, 129, 129 ), ( 129, 129, 129, 129 ), ( 129, 129, 129, 129 ) )
+       // CHECK:( ( 256, 256, 256, 256 ), ( 256, 256, 256, 256 ), ( 256, 256, 256, 256 ), ( 256, 256, 256, 256 ) )
```

Comment:
Is this different because the previous value was being "cut in half" by just walking through half of the tensor?

Reply:
This is different because batch size being considered was using the 0th dimension of packed A tensor, ignoring the last dimension (of size 2 in packed layout).